### ISSUE TITLE
KeyError: 'title'

### ISSUE THREAD

**Issue by TheFirstQuestion** [kiwiz/gkeepapi#41]
I'm trying out `gkeepapi` using the example code from the README, but I'm getting the following error:

```
Traceback (most recent call last):
  File "test.py", line 4, in <module>
    success = keep.login('*************@gmail.com', 'apppasswordhere')
  File "mydirectory/keepEnv/lib/python3.5/site-packages/gkeepapi/__init__.py", line 404, in login
    self.load(auth)
  File "mydirectory/keepEnv/lib/python3.5/site-packages/gkeepapi/__init__.py", line 418, in load
    self.sync(True)
  File "mydirectory/keepEnv/lib/python3.5/site-packages/gkeepapi/__init__.py", line 645, in sync
    self._parseNodes(changes['nodes'])
  File "mydirectory/keepEnv/lib/python3.5/site-packages/gkeepapi/__init__.py", line 672, in _parseNodes
    node = _node.from_json(raw_node)
  File "mydirectory/keepEnv/lib/python3.5/site-packages/gkeepapi/node.py", line 1450, in from_json
    node.load(raw)
  File "mydirectory/keepEnv/lib/python3.5/site-packages/gkeepapi/node.py", line 1027, in load
    self._title = raw['title']
KeyError: 'title'
```
I can see in my app passwords that this password has been used, so I'm assuming I am logging in successfully. I also tried [keepcli](https://github.com/mgckind/keepcli) and got the same error. 

Any solution? Thanks in advance.

**Comment by kiwiz** [comment]
It looks like Google is rolling out some changes to the data format. Can you try `print`ing out the value of `raw` at 1027 right before the exception is thrown?

**Comment by kiwiz** [comment]
Fixed in 0.11.1, thanks.
